app/javascript/article-form/components/ErrorList.jsx

```diff
@@ -10,6 +10,9 @@ export const ErrorList = ({ errors }) => {
       <h3 className="fs-l mb-2 fw-bold">Whoops, something went wrong:</h3>
       <ul className="list-disc pl-6">
         {Object.keys(errors).map((key) => {
+          if (key === 'base') {
```

Comment:
In response to the concerns that @jgaskins raised [here](https://github.com/forem/forem/pull/16704#discussion_r820952069), I decided to render the `base` word conditionally in the error messaging, instead of changing `:base` to `Errors` like I had done before.

I'm asking our frontend folks @aitchiss @nickytonline to weigh in on this change. The change I made is very localized, affecting only the error messaging on Post Edits, and only for error messages of the attribute `:base`. However, if there is a reason for rendering the `base` word that I am missing, please inform me 🙏🏾 

![Screen Shot 2022-03-08 at 10 46 19 AM](https://user-images.githubusercontent.com/32520970/157274267-8906b70b-0d3d-4128-9606-c121477f1097.png)

Reply:
It makes sense not to do it on the backend as per [Jamie's comment](https://github.com/forem/forem/pull/16704#discussion_r820952069). Is this the only place we need to do this? If not consider a utility function, e.g. `sanitizeError`.

Reply:
See above comment, but the reason for base ties to how Rails looks at errors (either on an attribute or on the "base" record) See https://guides.rubyonrails.org/active_record_validations.html#errors-base

Reply:
I like this approach and it preserves the idiomatic error key.

Reply:
(non blocking) We could make this a bit more succinct, e.g. 

```jsx
<ul className="list-disc pl-6">
  {Object.keys(errors).map((key) => {
    return (
      <li key={key}>
        {key === 'base' ? errors[key] : `${key}: ${errors[key]}`}
      </li>
    );
  })}
</ul>
```